Obedient-Lobsters/WorkforceManagement
# Computer: Delete an unassigned computer

# Description
Created functionally for a user to delete one unassigned computer. A deleteconfirm and a deleteconfirmed method was added to the computer controller, created DeleteConfirm and DeleteDenied in views folder and updated README
 Fixes #17 
This feature in conjunction with other functionality that is currently part of the master branch satisfies requirements for #11  
 ## Type of change
 Please delete options that are not relevant.
 # Testing Instructions

If you need to create the database:

Start SQL Server Management Studio

Run BangazonDatabaseCreateSeed.sql in SSMS

Update the appsetting.json in Visual Studio community to reflect your desktop and your server

If you have already created the database or has already completed the steps above:

Either:
 Seed a new computer in SMSS ```INSERT INTO Computer(DatePurchased, DateDecommissioned, Working, ModelName, Manufacturer) VALUES ( '2018/10/01',  null, 0, 'MacBook Pro','Apple');``` 
Or
Click on "Create New" link and complete the form to add a new computer once you have fetched and checkout the branch

In the terminal, `git fetch --all` and` git checkout ss-ComputerDelete` to switch branches

Start the program by cd'ing into the WorkforceManagement (make sure you are deep enough, the .sln file is a few WorkforceManagement directories deep) and using the command `start WorkforceManagement.sln '.

Press F5 to start the program

Click on the delete link on the computer index view. A delete confirm view with the computer details will be listed with a delete button.The view will show details regarding one computer. It will show id, date of purchase, date of decommissioning, functional status, model and manufacturer of the computer.

Click the delete button and if the computer has never been assigned, the index view will be presented without the deleted computer listed.

If the computer has ever been assigned then a delete denied view will populate with a link back to index. The Index view will show all computers including the computer that was attempted to be deleted

Read the README.md and check for completeness, grammar, and spelling.

 # Checklist:
